GitTfs/Core/TfsChangeset.cs

```diff
@@ -33,7 +33,7 @@ public LogEntry Apply(string lastCommit, IGitTreeModifier treeBuilder, ITfsWorks
             var resolver = new PathResolver(Summary.Remote, initialTree);
             var sieve = new ChangeSieve(_changeset, resolver);
             _changeset.Get(workspace, sieve.GetChangesToFetch(), ignorableErrorHandler);
-            foreach (var change in sieve.GetChangesToApply())
```

Comment:
Could you use an intermediate variable (called `forceGetChanges` or something more explicit) before calling the method?
It should be clearer?